sdk/resourcemanager/azure-resourcemanager-network/pom.xml

```diff
@@ -177,6 +177,7 @@
         <artifactId>revapi-maven-plugin</artifactId>
         <version>0.11.2</version> <!-- {x-version-update;org.revapi:revapi-maven-plugin;external_dependency} -->
         <configuration>
+          <skip>true</skip>
```

Comment:
Which API would get flagged in revapi?

Reply:
We need to allow defining outbound rule after defining public frontend, so the change is to let LoadBalancerFrontendImpl implements LoadBalancerPublicFrontend.Definition<LoadBalancer.DefinitionStages.WithCreateAndOutboundRule>
Below is the error:
[ERROR] java.method.returnTypeTypeParametersChanged: method com.azure.resourcemanager.network.models.LoadBalancerPublicFrontend.DefinitionStages.Blank<com.azure.resourcemanager.network.models.LoadBalancer.DefinitionStages.WithCreateAndOutboundRule> com.azure.resourcemanager.network.models.LoadBalancer.Definitio
nStages.WithPublicFrontend::definePublicFrontend(java.lang.String): The return type changed from 'com.azure.resourcemanager.network.models.LoadBalancerPublicFrontend.DefinitionStages.Blank<com.azure.resourcemanager.network.models.LoadBalancer.DefinitionStages.WithCreate>' to 'com.azure.resourcemanager.network.m
odels.LoadBalancerPublicFrontend.DefinitionStages.Blank<com.azure.resourcemanager.network.models.LoadBalancer.DefinitionStages.WithCreateAndOutboundRule>'. (breaks semantic versioning)

Reply:
Got it. This is fine.